src/layout/Footer.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  Link,
+  SeqeraIcon,
+  SlackIcon,
+  TwitterIcon,
+  YoutubeRectangleIcon
+} from 'website-components';
+
+const Footer = () => {
+  return (
+    <>
+      <footer className="bg-gray-900 text-white mt-auto">
+        <div className="container-lg">
+          <div className="border-white border-t opacity-40" />
+          <div className="flex justify-between py-4">
+            <div>
+              <Link to="https://seqera.io/" noBorder className="typo-small">
+                &copy; 2022 Seqera Labs
+              </Link>
+            </div>
+            <div className="flex items-center md:justify-end">
+              <div className="mr-4">
+                <Link to="http://twitter.com/hashtag/NextflowSummit" noBorder>
```

Comment:
This is bizarre and I don't understand why, but without the `?src` query string I get no results on the twitter link. Adding this seems to fix it.. 🤷🏻‍♂️ 

```suggestion
                <Link to="https://twitter.com/hashtag/NextflowSummit?src=hashtag_click" noBorder>
```